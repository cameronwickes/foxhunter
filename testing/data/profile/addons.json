{"schema":6,"addons":[{"id":"{randomID1}","icons":{"128":"https://addons.mozilla.org/random.png"},"type":"extension","name":"MI","version":"0.1.4","creator":{"name":"Cameron","url":"https://addons.mozilla.org/en-US/firefox/user/randomuser/"},"developers":[],"description":"MI","fullDescription":"A fake addon from Mozilla","screenshots":[{"url":"https://addons.mozilla.org/user-media/previews/full/random.png","width":920,"height":680,"thumbnailURL":"https://addons.mozilla.org/user-media/previews/full/random.png","thumbnailWidth":533,"thumbnailHeight":394}],"homepageURL":"https://mybrowseraddon.com/screen-recorder.html","supportURL":"https://mybrowseraddon.com/screen-recorder.html","contributionURL":"https://paypal.me/random","averageRating":3.5455,"reviewCount":7,"reviewURL":"https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/reviews/","weeklyDownloads":4366,"sourceURI":"https://addons.mozilla.org/firefox/downloads/file/random.xpi","updateDate":1614850516000},{"id":"{randomID2}","icons":{"128":"https://addons.mozilla.org/random.png"},"type":"extension","name":"NMI","version":"0.1.4","creator":{"name":"Cameron","url":"https://addons.mozilla.org/en-US/firefox/user/randomuser/"},"developers":[],"description":"NMI","fullDescription":"A fake addon not from Mozilla","screenshots":[{"url":"https://addons.mozilla.org/user-media/previews/full/random.png","width":920,"height":680,"thumbnailURL":"https://addons.mozilla.org/user-media/previews/full/random.png","thumbnailWidth":533,"thumbnailHeight":394}],"homepageURL":"https://mybrowseraddon.com/screen-recorder.html","supportURL":"https://mybrowseraddon.com/screen-recorder.html","contributionURL":"https://paypal.me/random","averageRating":3.5455,"reviewCount":7,"reviewURL":"https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/reviews/","weeklyDownloads":4366,"sourceURI":"https://randomsite.org/random.xpi","updateDate":1614850516000},{"id":"{randomID3}","icons":{"128":"https://addons.mozilla.org/random.png"},"type":"extension","name":"LDR","version":"0.1.4","creator":{"name":"Cameron","url":"https://addons.mozilla.org/en-US/firefox/user/randomuser/"},"developers":[],"description":"LDR","fullDescription":"A fake addon with low download rates from Mozilla","screenshots":[{"url":"https://addons.mozilla.org/user-media/previews/full/random.png","width":920,"height":680,"thumbnailURL":"https://addons.mozilla.org/user-media/previews/full/random.png","thumbnailWidth":533,"thumbnailHeight":394}],"homepageURL":"https://mybrowseraddon.com/screen-recorder.html","supportURL":"https://mybrowseraddon.com/screen-recorder.html","contributionURL":"https://paypal.me/random","averageRating":3.5455,"reviewCount":7,"reviewURL":"https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/reviews/","weeklyDownloads":10,"sourceURI":"https://addons.mozilla.org/firefox/downloads/file/random.xpi","updateDate":1614850516000},{"id":"{randomID4}","icons":{"128":"https://addons.mozilla.org/random.png"},"type":"extension","name":"LUR","version":"0.1.4","creator":{"name":"Cameron","url":"https://addons.mozilla.org/en-US/firefox/user/randomuser/"},"developers":[],"description":"LUR","fullDescription":"A fake addon with low user ratings from Mozilla","screenshots":[{"url":"https://addons.mozilla.org/user-media/previews/full/random.png","width":920,"height":680,"thumbnailURL":"https://addons.mozilla.org/user-media/previews/full/random.png","thumbnailWidth":533,"thumbnailHeight":394}],"homepageURL":"https://mybrowseraddon.com/screen-recorder.html","supportURL":"https://mybrowseraddon.com/screen-recorder.html","contributionURL":"https://paypal.me/random","averageRating":1.5455,"reviewCount":7,"reviewURL":"https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/reviews/","weeklyDownloads":4366,"sourceURI":"https://addons.mozilla.org/firefox/downloads/file/random.xpi","updateDate":1614850516000},{"id":"{randomID1}","icons":{"128":"https://addons.mozilla.org/random.png"},"type":"extension","name":"OOD","version":"0.1.2","creator":{"name":"Cameron","url":"https://addons.mozilla.org/en-US/firefox/user/randomuser/"},"developers":[],"description":"OOD","fullDescription":"A fake out of date addon from Mozilla","screenshots":[{"url":"https://addons.mozilla.org/user-media/previews/full/random.png","width":920,"height":680,"thumbnailURL":"https://addons.mozilla.org/user-media/previews/full/random.png","thumbnailWidth":533,"thumbnailHeight":394}],"homepageURL":"https://mybrowseraddon.com/screen-recorder.html","supportURL":"https://mybrowseraddon.com/screen-recorder.html","contributionURL":"https://paypal.me/random","averageRating":3.5455,"reviewCount":7,"reviewURL":"https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/reviews/","weeklyDownloads":4366,"sourceURI":"https://addons.mozilla.org/firefox/downloads/file/random.xpi","updateDate":1614850516000}]}