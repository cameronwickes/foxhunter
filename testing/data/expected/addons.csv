name,version,URL,storeURL,downloads,screenshots,rating
MI,0.1.4,https://addons.mozilla.org/firefox/downloads/file/random.xpi,https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/,4366,['https://addons.mozilla.org/user-media/previews/full/random.png'],3.5455
NMI,0.1.4,https://randomsite.org/random.xpi,https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/,4366,['https://addons.mozilla.org/user-media/previews/full/random.png'],3.5455
LDR,0.1.4,https://addons.mozilla.org/firefox/downloads/file/random.xpi,https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/,10,['https://addons.mozilla.org/user-media/previews/full/random.png'],3.5455
LUR,0.1.4,https://addons.mozilla.org/firefox/downloads/file/random.xpi,https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/,4366,['https://addons.mozilla.org/user-media/previews/full/random.png'],1.5455
OOD,0.1.2,https://addons.mozilla.org/firefox/downloads/file/random.xpi,https://addons.mozilla.org/en-US/firefox/addon/screen-recorder/,4366,['https://addons.mozilla.org/user-media/previews/full/random.png'],3.5455
